src/nvector/parallel/fmod/fnvector_parallel_mod.f90

```diff
@@ -906,20 +906,20 @@ function FN_VGetArrayPointer_Parallel(v) &
 use, intrinsic :: ISO_C_BINDING
 real(C_DOUBLE), dimension(:), pointer :: swig_result
 type(N_Vector), target, intent(inout) :: v
-type(C_PTR) :: fresult 
-type(C_PTR) :: farg1 
+type(C_PTR) :: fresult
+type(C_PTR) :: farg1
 
 farg1 = c_loc(v)
 fresult = swigc_FN_VGetArrayPointer_Parallel(farg1)
-call c_f_pointer(fresult, swig_result, [1])
+call c_f_pointer(fresult, swig_result, [FN_VGetLocalLength_Parallel(v)])
```

Comment:
`N_VGetLocalLength_Parallel` is missing from the `.h` and `.c` files

Reply:
Its there, perhaps the diff is only showing a subset of changes.

Reply:
https://github.com/LLNL/sundials/blob/main/include/nvector/nvector_parallel.h#L111 -- it already existed which is why its not in the diff

Reply:
Was looking for this in the diff but it already exists. All good.